InvenTree/stock/api.py

```diff
@@ -364,7 +364,7 @@ class Meta:
         ]
 
     # Relationship filters
-    manufactuer = rest_filters.ModelChoiceFilter(label='Manufacturer', queryset=Company.objects.filter(is_manufacturer=True), field_name='manufacturer_part__manufacturer')
+    manufacturer = rest_filters.ModelChoiceFilter(label='Manufacturer', queryset=Company.objects.filter(is_manufacturer=True), field_name='manufacturer_part__manufacturer')
```

Comment:
@SchrodingersGat I do not think that the change in this filter will break anything - am I overseeing anything?